crates/ironrdp-web/Cargo.toml

```diff
@@ -67,3 +70,4 @@ x509-cert = { version = "0.2", default-features = false, features = ["std"] }
 tap = "1"
 semver = "1"
 url = "2.4"
+base64 = "0.21"
```

Comment:
Suggestion: [base64ct](https://lib.rs/crates/base64ct) is a smaller alternative, probably a good idea for a WASM module.

Reply:
Well, `base64` API is better suited for our purpose, since we need to append base64 to an existing string (`base64::Engine::to_string` does just that), while `base64ct` either will always allocate a new string or could write into raw `&mut [u8]`). 

I think `base64` will work better here, but if you have a string opinion on that, I could switch the library

Reply:
Fair enough. I agree `base64` API is superior. Also, this is just a suggestion.

That being said, let me elaborate.
It’s possible to allocate a buffer of the right size in one go. For instance:
```rust
use base64ct::{Base64, Encoding as _};

const PREFIX: &str = "data:image/png;base64,";

let png_encoded_len = Base64::encoded_len(png_buffer);
let mut buffer = vec![0; PREFIX.len() + png_encoded_len];
buffer[..PREFIX.len].copy_from_slice(PREFIX);
Base64::encode(bytes, &mut buffer[PREFIX.len()..])?;

// buffer is holding the final result
```

I agree the code is less simple. However, the size of the resulting WASM module _is_ important too.
It would be a good idea to measure the impact of `base64` vs the impact of `base64ct` before making the move, making sure we indeed see a smaller artifact.
I think it’s okay to look into this separately and to open a follow-up PR.